app/packs/entrypoints/application.js

```diff
@@ -10,7 +10,8 @@
 import 'core-js/stable'
 import 'regenerator-runtime/runtime'
 
-require("@rails/ujs").start()
+import Rails from "@rails/ujs"
+Rails.start()
```

Comment:
in my local and production, I can see below error in chrome console

![image](https://github.com/user-attachments/assets/c5346816-dfdb-495e-9305-63c65c44d464)

using import to fix the error